plugin/pkg/scheduler/core/scheduling_queue.go

```diff
@@ -149,7 +149,7 @@ type UnschedulablePods interface {
 // scheduling. This is called activeQ and is a Heap. Another queue holds
 // pods that are already tried and are determined to be unschedulable. The latter
 // is called unschedulableQ.
-// Heap is already thread safe, but we need to acquire another lock here to ensure
+// Heap is not thread safe, so we need to acquire a lock here to ensure
```

Comment:
We could just remove this line and the one below. I was initially using a thread safe heap and that's why I was trying to explain why we needed to acquire another lock.

Reply:
Done.